style.css

```diff
@@ -0,0 +1,226 @@
+    background-repeat: no-repeat;
+    background-position: bottom;
+    width: 100%;
+    height: 200px;
+}
+
+.filler {
+    padding-top: 100px;
+}
+
+.navbar {
+    background-image: linear-gradient(#7f7f7f, #494949, #7f7f7f);
+    display: grid;
+    max-width: 900px;
+    margin: 0 auto;
+    height: 40px;
+    border-radius: 5px;
+    align-items: center;
+    justify-items: center;
+    grid-template-columns: 0.9fr 0.9fr 0.9fr 0.9fr 0.9fr 0.9fr 0.9fr 0.9fr 1.5fr;
+}
+
+.navbar a {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    text-decoration: none;
+    width: 100%;
+    text-align: center;
+    height: 100%;
+    border-right: 1px solid rgba(0, 0, 0, 0.5);
+    font-family: Roboto, sans-serif;
+    color: white;
+    font-size: 12px;
+}
+
+.navbar a:hover {
+    background-color: rgba(0, 0, 0, 0.5);
+    border-right: 1px solid rgba(0, 0, 0, 0.5);
+}
+
+.navbar>a>img {
+    height: 20px;
+    width: auto;
+}
+
+.navbar input {
+
+    background-image: url('images/search_icon.png');
+    background-repeat: no-repeat;
+    background-position: left 5px center;
+    background-size: 13px;
+
+
+
+    background-color: #7f7f7f;
+    width: 100px;
+    height: 18px;
+    border-radius: 50px;
+    border-top: 0px;
+    border-left: 0px;
+    border-right: 0px;
+    border-bottom: 1px solid rgba(255, 255, 255, 0.3);
+    box-shadow: inset 0px 6px 1px -4px rgba(0, 0, 0, 0.6);
+}
+
+.navbar input:focus {
+    background-image: url('images/search_icon.png');
+    background-repeat: no-repeat;
+    background-position: left 5px center;
+    background-size: 13px;
+
+
+    width: 200px;
+    padding-left: 25px;
+    margin-left: 10px;
+    margin-right: 10px;
+    transition-duration: 1s;
+
+}
+
+.text-container1 {
+    max-width: 1440px;
+    margin: 0 auto;
+    text-align: center;
+}
+
+.small {
+    padding-top: 60px;
+}
+
+.second {
+    margin-top: -13px;
+}
+
+.third {
+    background-image: url(https://web.archive.org/web/20140301004610im_/http://images.apple.com/home/images/cta_your_verse_46pt.png);
+    background-repeat: no-repeat;
+    background-position: top;
+    width: 100%;
+    height: 55px;
+}
+
+.third:hover {
+    background-image: url(https://web.archive.org/web/20140301004610im_/http://images.apple.com/home/images/cta_your_verse_46pt.png);
+    background-repeat: no-repeat;
+    background-position: bottom;
+    width: 100%;
+    height: 55px;
+}
+
+footer {
+    padding-top: 20px;
+    display: grid;
+    grid-template-columns: 1fr 1fr;
+    grid-template-rows: 1fr 1fr;
+    grid-gap: 10px;
+    max-width: 1000px;
+    margin: 0 auto;
+}
+
+.first-footer {
+    grid-column: 1;
+    grid-row: 1;
+    color: gray;
+    font-size: 13px;
+
+}
+
+.first-footer>small>a {
+    text-decoration: none;
+    color: blue;
+}
+
+.first-footer>small>a:hover {
+    text-decoration: underline;
```

Comment:
Here you are overriding the `text-decoration` from `none` to `underline`. But, there's no need to add `color: blue` as it is already inherting it.